envConfig/local.env

```diff
@@ -3,6 +3,7 @@ SIMORGH_PUBLIC_STATIC_ASSETS_ORIGIN=http://localhost:7080
 SIMORGH_PUBLIC_STATIC_ASSETS_PATH=/
 SIMORGH_ATI_BASE_URL=https://logws1363.ati-host.net?
 SIMORGH_EMBEDS_BASE_URL=https://polling.test.bbc.co.uk
+SIMORGH_EMBEDS_BASE_URL_OVERRIDE=https://polling.bbc.co.uk
```

Comment:
Would it be better for these to be called `SIMORGH_EMBEDS_BASE_URL_LIVE` and `SIMORGH_EMBEDS_BASE_URL_TEST` instead? As the override param allows us to pass test or live as an override

Reply:
I have updated the PR with these changes